### ISSUE TITLE
Scanning support?

### ISSUE THREAD

**Issue by philips77** [hellonuzzle/bluetooth-fct#1]
Hi, 
Have you add scanning support to the project? The documentation look quite outdated.

**Comment by hellonuzzle** [comment]
Yes, scanning support was added. A lot more has been added, actually. Need to update the documentation in general. We've been using this for production testing across hundreds of units so far and will be doing a 5k production build later this year using the automated tester. 

It's been invaluable for us!

**Comment by philips77** [comment]
Great to hear! So I'm waiting for an update :)
